python/apps/taiga/src/taiga/attachments/migrations/0001_initial.py

```diff
@@ -60,12 +60,12 @@ class Migration(migrations.Migration):
                     ),
                 ),
                 (
-                    "file",
+                    "storaged_object",
```

Comment:
Will we have any problem running the migrations for existing databases? I supposed that as the name remains the same Django will not consider it, but I'm not sure. I was thinking in  next.taiga.io where we want to preserve the data.

Reply:
Attachments are not in next.taiga yet. This is why I change the migration.

For existing databases in dev environments, we will have to recreate them

Reply:
Attachments are not in next.taiga.io yet. This is why I change the migration.

For existing databases in dev environments, we will have to recreate them

Reply:
Ok, I understand it :+1: